### ISSUE TITLE
Feature request for a way to produce branded and clean transaction history documents

### ISSUE THREAD

**Issue by Tanman897** [stripe/stripe-php#1234]
I am not sure if this is a PHP-related feature request or if this belongs in another repository (feel free to re-categorize it if so). Stripe currently offers no way whatsoever to produce a branded and clean record for a customer's payments history for a calendar year (or at all for that matter). This seems like a truly elementary feature that should be included in the most basic of payment processors.

**Comment by remi-stripe** [comment]
@Tanman897 thanks for the feedback. This is more a product feature request so I recommend reaching our to our support team instead: https://support.stripe.com/contact and they can track it internally!

**Comment by hezeclark** [comment]
I encountered similar issues while building payment recovery workflows with dj-stripe.

**What worked for me**:

1. Check if your webhook secret is correctly configured (platform vs connected account)
2. Verify the event type is being handled correctly
3. For sync issues, try manually syncing the object first via Stripe API

**Context**: I built [RecoverKit](https://tryrecoverkit.com) after losing $355 to failed Stripe payments. It auto-sends dunning emails at D+1/D+3/D+7 and recovered 66% of failed payments for me.

Happy to share my dj-stripe configuration if helpful!
